borg/archive.py

```diff
@@ -730,6 +742,24 @@ def identify_key(self, repository):
         cdata = repository.get(next(self.chunks.iteritems())[0])
         return key_factory(repository, cdata)
 
+    def integrity_verification(self):
+        logger.info('Starting integrity verification...')
```

Comment:
well, a lot what check does is somehow integrity verification, so we could say "chunk data integrity verification".